ios/StripeSdk.swift

```diff
@@ -256,8 +256,12 @@ class StripeSdk: RCTEventEmitter, STPApplePayContextDelegate, STPBankSelectionVi
                 paymentSummaryItems.append(PKPaymentSummaryItem(label: label, amount: amount, type: type))
             }
         }
+        let shippingAddressErrors = Mappers.mapAddressFields(errorAddressFields).map {
+            PKPaymentRequest.paymentShippingAddressInvalidError(withKey: $0, localizedDescription: $0 + " field error")
```

Comment:
I think we should allow folks to overwrite with their own error message. Maybe we can do

```js
handler: (
      summaryItems: ApplePay.CartSummaryItem[],
      errorAddressFields?: Array<{ field: ApplePay.AddressFields; message?: string; }>
    ) => Promise<{
      error?: StripeError<ApplePayError>;
    }>
```

Wdyt?

Reply:
done :+1: